code/.storybook/preview.tsx

```diff
@@ -280,7 +280,7 @@ const decorators = [
                 <div style={{ marginBottom: 20 }} />
               </>
             )}
-            <StoryFn />
```

Comment:
Can you explain?

Reply:
Sorry. This was just for testing purposes. I will revert. Good catch